uvcombine/uvcombine.py

```diff
@@ -973,9 +973,63 @@ def spectral_smooth_and_downsample(cube, kernelfwhm):
     return cube_ds_hdu
 
 
+try:
+    import dask
+    HAS_DASK = True
+except ImportError:
+    HAS_DASK = False
+
+if HAS_DASK:
+    import dask.array as da
+    from spectral_cube.dask_spectral_cube import add_save_to_tmp_dir_option
+
+    @add_save_to_tmp_dir_option
+    def _dask_feather_cubes(cube_hi, cube_lo,
+                            highresscalefactor=1.0,
+                            lowresscalefactor=1.0,
+                            weights=1.0,
+                            replace_hires=False,
+                            lowpassfilterSD=False,
+                            deconvSD=False):
+
+        lowresfwhm = cube_lo.beam.major
+
+        pixscale = wcs.utils.proj_plane_pixel_scales(cube_hi.wcs.celestial)[0]
```

Comment:
this pattern annoys me, so:
https://github.com/astropy/astropy/pull/13268

Reply:
I'll add an issue here for this as we use this EVERYWHERE

Reply:
#35